web/styles/app_components.css

```diff
@@ -492,6 +492,7 @@ div.overlay {
     @extend .stream-selection-header-colorblock;
     margin-bottom: 5px;
     z-index: 1;
+    min-width: 10px;
```

Comment:
Is this a bug fix for an issue affecting the move topic modal too? It might be helpful to make this its own commit with an explanation of the bug it's addressing.

Reply:
It's not a bug fix exactly, but directly related to the rest of this commit, to make sure it doesn't shrink when the stream name overflows. We want to make sure the stream name is cut off before the colorblock shrinks. I can add that to the commit message to be more clear if that helps.

Reply:
Okay I actually just double checked and I think some CSS fiddling I did since this removed the need for this CSS, so I'll just remove it! Simple solution.